src/Calendar.java

```diff
@@ -1,4 +1,168 @@
 
+import java.util.*;
+import java.awt.*;
+import javax.swing.table.*;
+import javax.swing.*;
+
 public class Calendar {
 
-}
+    // all the necessary components to make up Frame 1, the calendar component
+
+    static JComboBox navYear; // combobox to navigate thru the years
+    static Container container;
+    static JButton Prev, Next; //buttons for changing months
+    static JFrame frame;
+    static JLabel lMonth; //month label
+    static JLabel today;  //today's date
+    static JTable tCalendar; //table for the calendar view
+    static DefaultTableModel dtCalendar; //default tabel model
+    static JScrollPane spCalendar;
+    static JPanel panelCalendar;
+    static int theYear, theMonth, theDay, currYear, currMonth;
+
+
+    public static void main(String args[]) {
```

Comment:
Please make a constructor so that we can make an instance of the Calendar class in main.java and run the program from main.java. In the constructor, call a method populateFrame() and put all the code current in this main() method within there.